src/main/java/com/netchill/api/moviedb/services/serie/SerieApiService.java

```diff
@@ -0,0 +1,51 @@
+package com.netchill.api.moviedb.services.serie;
+
+import com.netchill.api.moviedb.MovieDBApiService;
+import com.netchill.api.moviedb.models.MovieDbPaginatedResponse;
+import com.netchill.api.moviedb.models.Production;
+import com.netchill.services.configuration.ConfigurationService;
+import okhttp3.MediaType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nullable;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Singleton
+public class SerieApiService {
+    // Logger pour ecrire des logs plusieur niveau .info .debug .error .warn
+    private static final Logger LOGGER = LoggerFactory.getLogger(SerieApiService.class);
```

Comment:
tu peux remplacer les logger par @Slj4 au dessus de ta class.
Ça va te générer un `this.log`